Source/WebCore/editing/DeleteSelectionCommand.cpp

```diff
@@ -404,6 +404,12 @@ bool DeleteSelectionCommand::initializePositionData()
     return true;
 }
 
+    // We don't want to inherit style from an element which don't have contents.
+    static bool shouldNotInheritStyleFrom(const Node& node)
+    {
+        return !node.canContainRangeEndPoint();
+    }
```

Comment:
This looks to be indented wrong. Maybe you have a smart editor that wants to indent everything in the WebCore namespace?